src/attorney.js

```diff
@@ -195,6 +196,18 @@ function applyArchiveConfig (config) {
   }
 }
 
+function applyArchiveFailedConfig (config) {
+  assert(!('archiveFailedAfterSeconds' in config) || config.archiveFailedAfterSeconds >= 1,
+    'configuration assert: archiveFailedAfterSeconds must be at least every second and less than ')
+
+  config.archiveFailedSeconds = config.archiveFailedAfterSeconds || config.archiveCompletedAfterSeconds
+  config.archiveFailedInterval = `${config.archiveFailedSeconds} seconds`
+
+  if (config.archiveFailedSeconds < 60) {
+    emitWarning(WARNINGS.CRON_DISABLED)
```

Comment:
Since this value can be defaulted from `archiveCompletedAfterSeconds`, we should not emit the warning twice.

Reply:
nice catch. addressed